static/app/views/issueDetails/groupDetails.tsx

```diff
@@ -474,6 +476,31 @@ function useLoadedEventType() {
   }
 }
 
+interface StreamlinedUIAnalyticsData {
+  fold_sections_open?: string;
+  sidebar_open?: boolean;
+}
+function useGetAnalyticsDataStreamlinedUI(): StreamlinedUIAnalyticsData {
+  const hasStreamlinedUI = useHasStreamlinedUI();
+  if (!hasStreamlinedUI) {
+    return {};
+  }
+
+  const sidebarOpenSections: Record<string, boolean> = {};
+  for (const sectionKey of Object.values(SectionKey)) {
+    const foldSectionKey = getFoldSectionKey(sectionKey);
+    const isOpen = localStorage.getItem(foldSectionKey) === 'true';
+    if (isOpen) {
+      sidebarOpenSections[`sidebar_${sectionKey}_open`] = true;
+    }
+  }
+
+  return {
+    sidebar_open: localStorage.getItem('issue-details-sidebar-open') === 'true',
+    ...sidebarOpenSections,
```

Comment:
doing it this way because we can't really send an array over and if we sent an individual event for each section open it would be too many events

Reply:
doesn't feel like this will be valuable to me since it also depends on if the page even has those sections

Reply:
oh its just the sidebar